specs/windows/routes.table

```diff
@@ -0,0 +1,17 @@
+table_name("routes")
```

Comment:
We shouldn't need this additional spec file for the `.table`, we _should just be able to pull the `posix/routes.table` file up one level and the code that generates the tables will pick it up.